code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -9,8 +9,7 @@
 	exotic_blood = "slimejelly"
 	damage_overlay_type = ""
 	var/datum/action/innate/regenerate_limbs/regenerate_limbs
-	toxic_food = MEAT | DAIRY
-	liked_food = TOXIC

```

Comment:
Wait why are you removing this

Reply:
I thought it made more sense for their preferred food to be what slimes actually eat rather than them liking all things poisonous to humans because their blood happens to be poisonous to humans

Reply:
>rather than them liking all things poisonous to humans because their blood happens to be poisonous to humans

Yeah but the actual reason was that slime people absorb matter from toxic reagents such as venom so logically, a slime person body would crave toxins just as humans crave salt.

Reply:
(No comments on why you can't use toxic reagents on normal slimes to feed them)